orchagent/intfsorch.cpp

```diff
@@ -1670,6 +1676,11 @@ bool IntfsOrch::isLocalSystemPortIntf(string alias)
 
 void IntfsOrch::voqSyncAddIntf(string &alias)
 {
+    if (m_singleVoq)
```

Comment:
Too many places in this PR checking for singleVoq vs multiVoq. Can we have a common logic to handle this? My concern is, more of such checks need to handled in future. @arlakshm, @abdosi please check.